org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/RenameResponse.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.services;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.lemminx.services.extensions.IRenameResponse;
+import org.eclipse.lsp4j.ResourceOperation;
+import org.eclipse.lsp4j.TextDocumentEdit;
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4j.VersionedTextDocumentIdentifier;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+
+public class RenameResponse implements IRenameResponse {
```

Comment:
Add Javadoc

Reply:
I can add JavaDoc, but maybe it's better to make it package-private (default) as this class should not be accessed from outside of lemminx (like `org.eclipse.lemminx.services.PrepareRenameRequest`)?

Reply:
+1

Reply:
Done